tests/importable_test.pyi

```diff
@@ -0,0 +1,21 @@
+from dataclasses import dataclass
+
+from _typeshed import Incomplete
+
+
+@dataclass(frozen=True)
+class EntryPointParam:
+    def __init__(self, group: str, name: str, spec: str) -> None:
+        self.group = group
+        self.name = name
+        self.spec = spec
+
+credential_plugins: Incomplete
```

Comment:
Why do you think it's incomplete? MyPy is perfectly capable of inferring the types the way they are set. Anyway, I don't think `*.pyi` are needed, as I mentioned in the other comment.